packages/text-field/test/text-field.test.js

```diff
@@ -159,74 +99,5 @@ describe('text-field', () => {
         expect(textField.hasAttribute('focused')).to.be.true;
       });
     });
-
-    describe('autoselect', () => {
-      it('default value of autoselect should be false', () => {
-        expect(textField.autoselect).to.be.false;
-      });
-
-      it('should not select content on focus when autoselect is false', async () => {
-        textField.value = '123';
-        input.dispatchEvent(new CustomEvent('focus', { bubbles: false }));
-        await aTimeout(1);
-        expect(input.selectionEnd - input.selectionStart).to.equal(0);
-      });
-
-      it('should select content on focus when autoselect is true', async () => {
-        textField.value = '123';
-        textField.autoselect = true;
-        await nextUpdate(textField);
-        input.dispatchEvent(new CustomEvent('focus', { bubbles: false }));
-        await aTimeout(1);
-        expect(input.selectionEnd - input.selectionStart).to.equal(3);
-      });
-    });
-  });
-
-  describe('has-value attribute', () => {
-    it('should toggle the attribute on value change', async () => {
-      textField.value = 'foo';
-      await nextUpdate(textField);
-      expect(textField.hasAttribute('has-value')).to.be.true;
-
-      textField.value = undefined;
-      await nextUpdate(textField);
-      expect(textField.hasAttribute('has-value')).to.be.false;
-    });
-
-    it('should not add the attribute when the value is an empty string', async () => {
-      textField.value = '';
-      await nextUpdate(textField);
-      expect(textField.hasAttribute('has-value')).to.be.false;
-    });
-
-    // User could accidentally set a 0 or false value
-    it('should add the attribute when the value is a number', async () => {
-      textField.value = 0;
-      await nextUpdate(textField);
-      expect(textField.hasAttribute('has-value')).to.be.true;
-    });
-
-    it('should add the attribute when the value is a boolean', async () => {
-      textField.value = false;
-      await nextUpdate(textField);
-      expect(textField.hasAttribute('has-value')).to.be.true;
-    });
-  });
-
-  describe(`methods`, () => {
```

Comment:
Both tests in this suite are covered here:

https://github.com/vaadin/web-components/blob/f67d18ca4f9a38e1798a513e99402916c0d6d148/packages/field-base/test/input-mixin.test.js#L32